impeller/tools/malioc_diff.py

```diff
@@ -277,6 +283,17 @@ def compare_shaders(malioc_tree, before_shader, after_shader):
 
 def main(argv):
   args = parse_args(argv[1:])
+
+  # Generate full paths if relative ones are provided.
+  args.before = (
+      args.before if os.path.isabs(args.before) else
+      os.path.join(BUILD_ROOT_DIR, args.before)
```

Comment:
Someone running the script by hand would expect that relative paths would be relative to their current working directory and not some other directory. Why does this have to be hardcoded to `BUILD_ROOT_DIR` and not the users cwd?

Reply:
The CI builds change the cwd context in multiple places which will make the execution of this script inconsistent depending of where is being run. From the build configuration file perspective we need a consistent way to define relative paths across generators and tests that need to access code, out and third_party dependencies.

Reply:
The build configurations can't resolve to full paths as all the builds run from temporary sandboxes and real paths change all the time. Any specific names that you would like for the additional params?

Reply:
I'm thinking on using `before-relative-to-checkout` and `after-relative-to-checkout`

Reply:
Done